.travis.yml

```diff
@@ -25,11 +36,12 @@ services:
 before_install:
     - sudo add-apt-repository -y ppa:duggan/bats
     - sudo apt-get update
-    - sudo apt-get -qq install bats btrfs-tools git libapparmor-dev libc-dev libdevmapper-dev libglib2.0-dev libgpgme11-dev libselinux1-dev
+    - sudo apt-get -qq install bats btrfs-tools git libapparmor-dev libc-dev libdevmapper-dev libglib2.0-dev libgpgme11-dev libselinux1-dev realpath
     - sudo apt-get -qq remove libseccomp2
     - sudo apt-get -qq update
     - sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce
     - mkdir /home/travis/auth
+    - sudo mkdir -p /var/lib/containers/storage/overlay
```

Comment:
hmm,  the lack of this prior might be the reason for some errors I'm seeing in another PR that' I've been slowly picking at.

Reply:
Ack, I added this to work around buildah operations in the Fedora container, earlier it complained about overlay not supported